openerp-atomfeed-service/src/main/java/org/bahmni/feed/openerp/domain/OpenMRSResource.java

```diff
@@ -0,0 +1,52 @@
+package org.bahmni.feed.openerp.domain;
+
+import java.util.Date;
+
+public class OpenMRSResource {
+
+    private String id;
```

Comment:
the "id" is actually the "uuid" of the resource. Maybe name the variable as "uuid" and use @ JsonProperty("id") over the setter method